lib/gitter-adapter.js

```diff
@@ -148,6 +148,14 @@ Adapter.prototype.sendMessage = function(target, message) {
   var uri = target.replace('#','');
   var isStatus = /^\u0001ACTION/.test(message);
 
+  var regResult = /^((\w+)[:,]\s*).+/.exec(message);
+  var mention, mentioned;
+
+  if (regResult !== undefined) {
```

Comment:
This should probably be `!== null`. `.exec` that produces no match returns a `null`.